### ISSUE TITLE
Remove links to missing Figma libraries

### ISSUE THREAD

**Issue by danbrady** [department-of-veterans-affairs/vets-design-system-documentation#2150]
## Description
Currently the Figma DS has a reference to a missing library:

![Screenshot 2023-10-03 at 9 20 32 AM](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/381190/ffe14598-1386-4caa-942c-091cd9ed62cb)

This is likely from when the previously created components were imported.

## Tasks
- [x] As imported components are revisit, be sure they don't reference any external libraries (for colors, icons, etc.)

## Acceptance Criteria
- [x] Validate there are no links to missing libraries in the Figma DS

**Comment by danbrady** [comment]
Hallelujah! I think I finally solved this. For any future issues regarding phantom missing libraries, I was able to use the method in the answer found here: [Finding Missing Library Components](https://forum.figma.com/t/finding-missing-library-components-styles/6681/1).

@caw310 This wasn't in this sprint but stumbled upon the fix. Let me know if I need to take further action in order to close it out correctly.
